saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/property/ClassifierContainsInLBP.scala

```diff
@@ -0,0 +1,38 @@
+/** This software is released under the University of Illinois/Research and Academic Use License. See
+  * the LICENSE file in the root folder for details. Copyright (c) 2016
+  *
+  * Developed by: The Cognitive Computations Group, University of Illinois at Urbana-Champaign
+  * http://cogcomp.cs.illinois.edu/
+  */
+package edu.illinois.cs.cogcomp.saul.datamodel.property
+
+import java.util
+
+import edu.illinois.cs.cogcomp.lbjava.classify.{ Classifier, FeatureVector }
+
+private[property] class ClassifierContainsInLBP[T](val property: Property[T]) extends Classifier {
```

Comment:
Just noticed that there was another file with this name. Still, maybe we can choose a better name? (I'm brain dead now)